app/src/store/pointCloudStore.ts

```diff
@@ -605,9 +599,9 @@ export const createPointCloudStore = (initProps?: Partial<PointCloudProps>) => {
     dimension: null,
     dimensionMetadata: null,
     umapParameters: {
-      minDist: DEFAULT_MIN_DIST,
-      nNeighbors: DEFAULT_N_NEIGHBORS,
-      nSamples: DEFAULT_DATASET_SAMPLE_SIZE,
+      minDist: window.Config.minDist,
+      nNeighbors: window.Config.nNeighbors,
+      nSamples: window.Config.nSamples,
```

Comment:
I think it might be a tad cleaner if we leave the constants imports and place the window defaults within that file.